### ISSUE TITLE
Loss function misses the latenet regularization part?

### ISSUE THREAD

**Issue by fishfishson** [amosgropp/IGR#1]
Dear author, 

The loss function in your train.py misses the latent regularization part. In my opinion, it should be added to the variable loss. Besides the latent lambda is 1e-3 not 1e-2 in config file, which is incompatible with the setting in your paper, is there trick in this setting?

Thx

**Comment by amosgropp** [comment]
Indeed this is a bug - we did train it with the latent loss it was just slipped in this version.
The affect of the latent lambda is not that important we tried with both 1e-2 and 1e-3 - the specific pretrained model that we uploaded was trained with lambda 1e-2 but we tried with both.
Sorry for the problem we will fix it right ahead! Thanks for the awareness!

**Comment by amosgropp** [comment]
fixed it
